src/axom/spin/BVH.hpp

```diff
@@ -195,40 +201,40 @@ class BVH
    * \brief Returns the scale factor used when constructing the BVH.
    * \return scale_factor the scale factor
    */
-  FloatType getScaleFacor() const { return m_scaleFactor; };
+  FloatType getScaleFactor() const { return m_scaleFactor; };
 
   /*!
    * \brief Sets the tolerance used for querying the BVH.
    * \param [in] TOL the tolerance to use.
    *
    * \note Default tolerance set to floating_point_limits<FloatType>::epsilon()
    */
-  void setTolerance(FloatType TOL) { m_Tolernace = TOL; };
+  void setTolerance(FloatType TOL) { m_tolerance = TOL; };
```

Comment:
Consistency (minor): We use `EPS` for the tolarance variable in most of `primal`. It might be good to make the change here for consistency throughout `axom`

```suggestion
  void setTolerance(FloatType EPS) { m_tolerance = EPS; };
```